src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -22,17 +22,30 @@
 #include <ripple/app/main/Application.h>
 #include <ripple/app/misc/Validations.h>
 #include <ripple/app/data/DatabaseCon.h>
+#include <ripple/core/ConfigSections.h>
 #include <boost/format.hpp>
+#include <boost/tokenizer.hpp>
 
 namespace ripple {
 
+namespace detail
+{
+/** Amendments that are "compiled into" this app
+
+    Add amendments to this collection at build time to enable them on this
+   server. The first element of the pair is the id, the second element is
+   the "friendly name".
+*/
+std::set<std::pair<std::string, std::string>> PreEnabledAmendments;
+}
+
 /** Track the list of "amendments"
 
     An "amendment" is an option that can affect transaction processing
```

Comment:
I don't think it's a lot of work. I'm arguing that a more focused pull request is better. So this pull request adds amendments (and admittedly did some local cleanup while I was there). If I change to tagged uint256 there non-local cleanups that seem too unrelated to this pull request.

I understand we need to balance extra paperwork (create new ticket, create new pull request w/ reviews). At any rate, not a lot of work. Go head and do it?